### DISCUSSION TITLE
为什么vue组件访问不了外部http 的api

### DISCUSSION THREAD

**Discussion by mingruankeji** [alex8088/electron-vite#354] [Q&A] [answered]
用npm create @quick-start/electron创建的vue项目.为什么vue组件访问不了外部http 的api.主进程访问可以,渲染进程访问就报错:Home.vue:58 Refused to connect to 'http://127.0.0.1:22111/api/SampleData/Sgettbzhjl' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'connect-src' was not explicitly set, so 'default-src' is used as a fallback.

**Comment by mingruankeji** [comment, marked as answer]
我自己找到原因了,感谢,结贴.

**Reply by tahitimoon** [reply]
> 我自己找到原因了,感谢,结贴.

什么原因了？请求外部api，你是放在渲染层还是主进程？
